### DISCUSSION TITLE
[2021.08.23]  완주하지 못한 선수

### DISCUSSION THREAD

**Discussion by pyunji** [pyunji/Hyundai-ITnE-Team_QuadCore#21] [Algorithm]
[완주하지 못한 선수](https://programmers.co.kr/learn/courses/30/lessons/42576)

**Comment by pyunji** [comment]
[완주하지 못한 선수](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/PR/PR_42576.java)

**Comment by chaeyeon4u** [comment]
[완주하지 못한 선수](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/Java/PR_42576.java)

**Comment by leechang4** [comment]
[완주하지 못한 선수](https://github.com/leechang4/Hyundai-ITnE-Team_QuadCore/blob/master/lch/Programmers/PR_42576.java)

**Comment by hyk339** [comment]
[solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/commit/28036ea186e1f224b2f47fcc56a82044f79945a9)
